NeuralAmpModeler/NeuralAmpModeler.cpp

```diff
@@ -618,6 +630,24 @@ void NeuralAmpModeler::ProcessBlock(iplug::sample** inputs, iplug::sample** outp
   // restore previous floating point state
   std::feupdateenv(&fe_state);
 
+  // IR Toggle
+  const bool IRToggleIsActive = this->GetParam(kIRToggle)->Value();
+  if (IRToggleIsActive)
+  {
+    if (this->mIR == nullptr && this->mIRPath.GetLength())
+      this->_GetIR(this->mIRPath);
```

Comment:
Ooh. This is the RT function; I would have expected not to see anything in here except a change to lines 627 i.e.

```cpp
if (this->mIR != nullptr && this->GetParam(kIRToggle)->Value())
```
and avoid doing things to the GUI if at all possible.

I'd think it'd be possible to do something just like the other theme-switching that the EQ & gate switches do?

Reply:
nice, i didnt see this.